collivery.php

```diff
@@ -183,4 +183,15 @@ function mds_change_default_checkout_country()
 
         add_filter('default_checkout_country', 'mds_change_default_checkout_country');
     }
+    
+    add_action( 'woocommerce_view_order', 'before_woocommerce_order_details', 5 );
+    function before_woocommerce_order_details($order_id){
+        global $wpdb;
+        $table_name = $wpdb->prefix.'mds_collivery_processed';
+        $data_ = $wpdb->get_results('SELECT * FROM `'.$table_name.'` WHERE order_id='.$order_id.';', OBJECT);
+        $mds = MdsColliveryService::getInstance();
+        $waybillId = (int)$data_[0]->waybill;       
```

Comment:
This might cause the customer to be throttled, not sure what the limits are

Reply:
You're right, each time the page load its a request :|